api/pb/api.proto

```diff
@@ -97,39 +103,15 @@ service SpacemeshService {
           body: "*"
         };
     }
-    rpc SetCommitmentSize(CommitmentSizeMessage) returns (SimpleMessage) {
-        option (google.api.http) = {
-          post: "/v1/setcommitmentsize"
-          body: "*"
-        };
-    }
-    rpc SetLogicalDrive(LogicalDriveMessage) returns (SimpleMessage) {
-        option (google.api.http) = {
-          post: "/v1/setlogicaldrive"
-          body: "*"
-        };
-    }
     rpc SetAwardsAddress(AccountId) returns (SimpleMessage) {
         option (google.api.http) = {
           post: "/v1/setawardsaddr"
           body: "*"
         };
     }
-    rpc GetInitProgress(google.protobuf.Empty) returns (SimpleMessage) {
-        option (google.api.http) = {
-          post: "/v1/getprogress"
-          body: "*"
-        };
-    }
-    rpc GetTotalAwards(google.protobuf.Empty) returns (SimpleMessage) {
-        option (google.api.http) = {
-          post: "/v1/gettotalawards"
-          body: "*"
-        };
-    }
-    rpc GetUpcomingAwards(google.protobuf.Empty) returns (EligibleLayers) {
+    rpc GetMiningStats(google.protobuf.Empty) returns (MiningStats) {
         option (google.api.http) = {
-          post: "/v1/getupcomingawards"
+          post: "/v1/stats"
```

Comment:
mining-stats ?

Reply:
let leave it as is for now, there are no other stats or other stats can be added to this rpc later on